(DOCS) Add release notes for 6.11.0 (#3271)
puppetlabs/puppetdb: documentation/release_notes/release_notes_latest.markdown

---

## PuppetDB 6.11.0

### Upgrading

We recommend upgrading to PostgreSQL 11 or greater before upgrading to
PuppetDB 6.11.0. PostgreSQL 11 includes performance improvements which make
adding a non-null column with a default value much faster and should
significantly speed up the migration included in this release.

### New features

>  - **Added support for storage of reports generated by Bolt Plans** Bolt Plans will start submitting reports at a later date

Reviewer: ```suggestion
 - **Support for the storage of reports generated by Bolt Plans**.
```
I don't think we usually mention future features.

>  - **New report field `type`. Specifies the type of report submitted.** Either 'agent' for a puppet agent run, or 'plan' for a report of a plan's apply block

Reviewer: ```suggestion
 - **New report field `type` to specify the type of report submitted.** Use `agent` for a Puppet agent run, or `plan` for a report of a plans apply block
```

 - #TODO figure out what we want to mention about command broadcast

> ### Notes

Reviewer 2: ```suggestion
### Deprecations
```
I feel like people might skip reading the notes

Author: 👍 updated

>    - Java 8, 9, and 10 have been deprecated and PuppetDB will now log warnings on startup if these version of Java are being used. We recommend using Java 11 going forward

Reviewer: ```suggestion
   - Java 8, 9, and 10 have been deprecated. If these versions are used, PuppetDB will log a warning on startup. We recommend using Java 11 going forward.
```

>    - Running PuppetDB with PostgreSQL 9.6 and 10 has been deprecated, please use PostgreSQL 11

Reviewer: ```suggestion
   - Running PuppetDB with PostgreSQL 9.6 and 10 has been deprecated. Use PostgreSQL 11 instead. 
```
